Dockerfile-python

```diff
@@ -1,5 +1,23 @@
-FROM python:3
-RUN pip install postgres
-RUN pip install psycopg2
+FROM python:3.6
+COPY requirements.txt /
+RUN pip install -r /requirements.txt
 RUN apt update
-RUN apt install -y postgresql 
\ No newline at end of file
+RUN apt install -y postgresql 
+
+RUN apt-get update \
+  && apt-get -y upgrade \
+  && apt-get -y install git build-essential libsqlite3-dev zlib1g-dev
+
+# Create a directory and copy in all files
+RUN mkdir -p /tmp/tippecanoe-src
+RUN git clone https://github.com/mapbox/tippecanoe.git /tmp/tippecanoe-src
+WORKDIR /tmp/tippecanoe-src
+
+# Build tippecanoe
+RUN make install
+
+# Remove the temp directory and unneeded packages
+WORKDIR /
```

Comment:
ideally `workdir` should be next statement after `from` stmt. Other commands can be done following to that.